conda_build_config.yaml

```diff
@@ -0,0 +1,30 @@
+c_compiler:         # [win]
+  - vs2008          # [win]
+  - vs2015          # [win]
+cxx_compiler:       # [win]
+  - vs2008          # [win]
+  - vs2015          # [win]
+hdf5:
```

Comment:
We likely will need to maintain packages for hdf5 1.8 for a while longer.  1.10 introduced format breaking changes that AFIAK some downstream projects (e.g. netcdf) have not yet determined how to deal with.